.rubocop.yml

```diff
@@ -7,10 +7,10 @@ AllCops:
 
   DisplayCopNames: true
 
-Metrics/LineLength:
+Layout/LineLength:
   Max: 120
 Metrics/MethodLength:
-  Max: 20
+  Max: 40
```

Comment:
- [ ] As the last code reviewer said it's mandatory to stick with the Linters law. Metrics/MethodLength should be Max: 20